### ISSUE TITLE
Panel giving "whoops" error after logging in using Edge

### ISSUE THREAD

**Issue by xyeLz** [Pterodactyl/Panel#747]
I rarely use Edge as a browser but simply needed to test something. After authenticating successfully, I was directed to the whoops page with the following errors in the logs:

`Next Doctrine\DBAL\Driver\PDOException: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'sql01_db01.tasks' doesn't exist in /mnt/var/www/html/pterodactyl/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:79

It seems to be fine in IE, Chrome, and Firefox, but wanted to mention this since I spent a couple hours troubleshooting what apparently doesn't seem to be an actual installation issue.

**Comment by lancepioch** [comment]
Version of Panel?

**Comment by xyeLz** [comment]
Updated above. Sorry. I put this in the original message but cleared my cache and GitHub logged me out.

**Comment by xyeLz** [comment]
FWIW, this happens on both of my installations in completely separate environments. Do you have an environment I could test using Edge to verify if it's just my environments for some reason? Additionally, do the logs I provided mean anything that could resolve this?

**Comment by DaneEveritt** [comment]
I don't see how a specific browser would trigger a server specific error.

**Comment by xyeLz** [comment]
I mean I don't know what you want me to say. I obviously don't disagree that the behavior is unusual. I'm just reporting my findings. If you have a test environment and would like me to test it using Edge (not that you can't simply test it yourself), I would be happy to do so. Obviously my panel installation isn't the same as others, so I don't know if my findings are dissimilar to everyone else, but I wanted to provide you with my findings and the logs just in case I'm not the first or only one having the problem.

**Comment by DaneEveritt** [comment]
If you could at least provide the whole stack trace for the error that would be helpful here. I can download a test image and see, but I don't even know why it would be looking for the tasks table like that.

**Comment by xyeLz** [comment]
Yeah of course let me look into it. I just tested again using Edge and while I'm still getting the "whoops" error, I'm not seeing anything new show up in the storage/logs folder after deleting today's log. Is there somewhere else I could look?

**Comment by DaneEveritt** [comment]
If you get a whoops page, it is in the laravel logs. If it is not, I have no idea because I've never seen that happen.

**Comment by xyeLz** [comment]
Yeah I have no idea. I'm getting the error but no logs are being generated. I guess I'll just respond on this chain if I can reproduce it again...
